code/src/components/UserMail.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable jsx-a11y/no-noninteractive-tabindex */
+import React from 'react';
+
+const UserMail = ({ usermail, setUsermail }) => {
+  const handleMailInputChange = (event) => {
+    setUsermail(event.target.value);
+  };
+  return (
+    <div>
+      <p className="user-p" tabIndex="0">What is your email address?</p>
+      <input value={usermail} className="user-input" type="email" placeholder="email address" required="required" onChange={handleMailInputChange} />
```

Comment:
I'm not sure why, but it doesn't seem like the `type="email"` is working properly. I'm able to go to the next questions without adding a valid email address with a "@" and ".com". Perhaps worth looking into?